tests/unit/components/CombinationMenuItem-test.js

```diff
@@ -0,0 +1,237 @@
+            // How do I check my data() to make sure this.children is correct?
+            // expect(wrapper.vm.children).to.equal([])
+            // expect(spy).to.have.been.called()
+            done() // ends the test
+          })
+    })
+
+    // it('can $post to get children', done => {
+    //     const imageID = 4284
+    //     const plate = '1094'
+    //     const fragment = '1'
+    //     const institution = 'IAA'
+    //     const scrollVersionID = '1231'
+    //     const versionID = scrollVersionID
+    //     const scrollID = '894'
+
+    //     const returnData = {
+    //         status: 200,
+    //         data: {
+    //             results: [
+    //                 {artefact_id: 1},
+    //             ],
+    //         },
+    //       }
+
+    //     // create wrapper with mocked route and routers
+    //     let wrapper = mount(ImageMenuItem, {
+    //         propsData: {
+    //             dataId: imageID,
+    //             plate: plate,
+    //             fragment: fragment,
+    //             institution: institution,
+    //             versionID: versionID,
+    //         },
+    //     })
+    //     wrapper.vm.$post = function() {
+    //         return new Promise((resolve, reject) => resolve(returnData))
+    //     }.bind(vm)
+
+    //     wrapper.vm.fetchChildren()
+
+    //     // assertions
+    //     wrapper.vm.$nextTick(() => {
+    //         // How do I check my data() to make sure this.children is correct?
+    //         // expect(wrapper.vm.children).to.deep.equal(returnData.data.results)
+    //         done() // ends the test
+    //       })
+        
+    // })
+
+    // it('pushes data to the router', () => {
+    //     const push = sinon.spy()
+    //     const imageID = 4284
+    //     const plate = '1094'
+    //     const fragment = '1'
+    //     const institution = 'IAA'
+    //     const scrollVersionID = '1231'
+    //     const versionID = scrollVersionID
+    //     const scrollID = '894'
+
+    //     // create wrapper with mocked route and routers
+    //     let wrapper = mount(ImageMenuItem, {
+    //         propsData: {
+    //             dataId: imageID,
+    //             plate: plate,
+    //             fragment: fragment,
+    //             institution: institution,
+    //             versionID: versionID,
+    //         },
+    //         mocks: { 
+    //             $router: { push },
+    //             $route: {
+    //                 params: { 
+    //                     imageID,
+    //                     scrollID: scrollID,
+    //                     scrollVersionID: scrollVersionID,
+    //                 }
+    //             }
+    //         }
+    //     })
+
+    //     wrapper.vm.setRouter()
+
+    //     // assertions
+    //     expect(push.firstCall.args[0].name).to.equal("workbenchAddress")
+    //     expect(push.firstCall.args[0].params).to.include({ imageID })
+    // })
+
+    it('has a span', () => {
```

Comment:
Again, I'm wondering about these level of super generic tests? Do you feel like they're worth-while? Can they become something more specific?